en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+# PrivateJet mode for Microgateways
+With many of applications gearing towards microservice architecture, it’s no surprise that container-orchestration systems such as Kubernetes have become so popular with functionalities such as automating computer application deployment, scaling, and management, 
+which simplifies a number of complex management tasks. WSO2 API-Manager provides cloud-native API management where user can expose microservices as managed APIs in cloud environments such as Kubernetes with the support of [WSO2-Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md). 
+
+ ![Architecture]({{base_path}}/assets/img/learn/privatejet-mode/architecture.png)
+ 
+Microservices will expose as managed APIs in cloud clusters in privetJet mode, where each microservice will have a dedicated [WSO2 API Microgateway](https://wso2.com/api-management/api-microgateway/). This will provide maximum security and guaranteed resource allocation for API execution. As depicted in the above diagram, When the APIs published via API-Manager in cloud environments,
+deployment, scaling, and management tasks will handle by the [WSO2 -Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md) itself.
+
+## Configuring APIs to deploy in PrivateJet mode
+
+Follow the steps below to configure the PrivateJet mode for Microgateways in API Manager to deploy APIs as managed APIs in cloud clusters. (Ex: Kubernetes)
+
+1. Install and setup [Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md). In order to expose APIs using ingress controller [deploy the k8s-api-operator in ingress mode](https://github.com/wso2/k8s-api-operator/tree/v1.1.0/scenarios/scenario-17). 
+2. Deploying cluster configurations
+ 
+    Processes in containers inside pods are authenticated when contacting the apiserver and accessing resources. In order to regulate access to the resources need to establish the RBAC Authorization with at least permissions to, 
+    Create, Delete, Update, Get , List, Post the customresourcedefinitions, apis and configmaps.
+
+    i.  Create a service account
+
+        kubectl create serviceaccount <service-account name> -n <namespace>
+
+    ii. Create a clusterRole
+
+        apiVersion: rbac.authorization.k8s.io/v1
+        kind: ClusterRole
+        metadata:
+          name: clusterrole-privatejet
+        rules:
+        - apiGroups: ["","apiextensions.k8s.io","wso2.com"]
+          resources: ["configmaps","customresourcedefinitions","apis"]
+          verbs: ["get", "post", "create", "delete", "put", "list","update"]
+     
+    iii. Create a  clusterRoleBinding
+
+        apiVersion: rbac.authorization.k8s.io/v1
+        kind: ClusterRoleBinding
+        metadata:
+          name: clusterrolebinding-privatejet
+        subjects:
+        - kind: ServiceAccount
+          name: wso2am-privatejet # Name is case sensitive
```

Comment:
wso2am-privatejet is the service account we created in step 1.

Reply:
done